docs/NoSSL/lavalink-without-ssl.md

```diff
@@ -234,3 +234,12 @@ Port : 30216
 Password : "somboytiger"
 Secure : false
 ```
+
+### Hosted by @ [𝓓𝓔𝓛𝓤𝓢𝓘𝓞𝓝](https://discord.gg/FqEQtEtUc9)
+Version 3.?
```

Comment:
What version is it? 3.What? 🙁

Reply:
if i not wrong, i think he still used version `yt-400-v3`.
let me ask him

Reply:
yes you correct (219dfdd01931edd3d750e2757de23aa2d228b9c9-SNAPSHOT)

Reply:
updated